docs/api/v1.md

```diff
@@ -106,12 +292,89 @@ to stream trace with chunked encoding, maybe omit status).
 }
 ```
 
+
 ### Execution
 ```
 {
-    id {int}: Unique Identifier
-    otherInfo {object}: TODO Whatever other info, ask Russell 
+   id {integer}: Unique Identifier
+   type {string}: Type of execution. Either `function` or `process`
+   status {string}: Either `pending`, `executing`, `stopped`, or `done`
+   executionTime {integer}: Nanoseconds
+   data {object}: Data specific to this type
+      if `type=function`: {
+         function {[Function](#function)}: Function that produced this execution
```

Comment:
Isn't this redundant since anyone receiving a function execution is also the one who started it? I suppose it fits in with the server-side execution caching, but I'd rather we avoid leaving that as an accidental crutch for the frontend.